content/200-orm/500-reference/200-prisma-cli-reference.mdx

```diff
@@ -130,13 +126,13 @@ The `init` command does not interpret any existing files. Instead, it creates a
 
 | Argument                | Required | Description                                                                                                                                                                                                                         | Default                       |
 | ----------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- |
-| `--datasource-provider` | No       | Specifies the value for the `provider` field in the `datasource` block. Options are `sqlite`, `postgresql`, `mysql`, `sqlserver`, `mongodb` and `cockroachdb`.                                                                      | `postgresql`                  |
+| `--datasource-provider` | No       | Specifies the value for the `provider` field in the `datasource` block. Options are `prisma+postgres`, `sqlite`, `postgresql`, `mysql`, `sqlserver`, `mongodb` and `cockroachdb`.                                                   | `postgresql`                  |
+| `--db`                  | No       | Shorthand syntax for `--datasource-provider prisma+postgres`; creates a new [Prisma Postgres](/orm/overview/databases/prisma-postgres) instance. Requires authentication in the [PDP Console](https://console.prisma.io).           |                               |
```

Comment:
Probably need clarification from @luanvdw 

Are we saying PDP console or Prisma Console?